### PR TITLE
Fix/update waf rules for gca bounce rate

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

This PR adds 4 new paths to cover 2 new content pages being added as part of the bounce rate work:
1. `/keep-accurate-contact-information`
2. `/maintenez-a-jour-les-coordonnees`
3. `/delivery-and-failure`
4. `/livraison-reussie-et-echec`

### CODE REVIEW

File: .github/workflows/infrastructure_version.txt [hunk 1]
```diff
@@ -1 +1 @@
-1.2.11
+1.2.14
```
Comment (jimleroyer) [inline .github/workflows/infrastructure_version.txt:1]: There is a conflict on that file atm. We should make a release in a separate PR excluding the changes too (that gives time to test the changes in staging).

Comment (andrewleith) [reply .github/workflows/infrastructure_version.txt]: All fixed up.  No rush on this I dont think...

**Review by jimleroyer — APPROVED** [review]
LGTM!
